server/controllers/list.js

```diff
@@ -0,0 +1,46 @@
+const List = require('../models/List')
+const asyncHandler = require('express-async-handler')
+
+//add a list to the collection
+//Route POST /list/addList
+exports.addList = asyncHandler(async (req, res, next) => {
+  List.create(req.body).then(function (list) {
+    res.send(list)
+  })
```

Comment:
There should be a linkage established here with the logged in user

Reply:
by linkage do you mean that each user object should have an array of lists and when I add a list that array should be updated on the user object? Currently my models are such that lists have the ObjectId of the user which is required.

Reply:
Neither of those are happening here though, it's just creating a list. Unless you're passing the userId as part of the request body, which is a security concern as any user could create a list on someone else's account